base/linalg/bunchkaufman.jl

```diff
@@ -43,14 +43,12 @@ end
 """
     bkfact(A, rook::Bool=false) -> BunchKaufman
 
-Compute the Bunch-Kaufman [^Bunch1977] factorization of a symmetric or Hermitian matrix `A` as ``PUDU'P`` or ``PLDL'P``, depending on which triangle is stored in `A`, and return a `BunchKaufman` object.
+Compute the Bunch-Kaufman [^Bunch1977] factorization of a symmetric or Hermitian matrix `A` as ``P'*U*D*U'*P`` or ``P'*L*D*L'*P``, depending on which triangle is stored in `A`, and return a `BunchKaufman` object. If `A` is complex symmetric then `U'` and `L'` is used as notation for the unconjugated transposes, i.e. `Transpose(U)` and `Transpose(L)`.
```

Comment:
Perhaps "... then `U'` and `L'` denote the unconjugated transposes, i.e. `Transpose(U)` and `Transpose(L)`"?